handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/LoadBalancer.java

```diff
@@ -58,7 +58,9 @@ public ServiceCombServer chooseServer(Invocation invocation) {
     List<ServiceCombServer> servers = invocation.getLocalContext(LoadbalanceHandler.CONTEXT_KEY_SERVER_LIST);
     int serversCount = servers.size();
     for (ServerListFilterExt filterExt : filters) {
-      servers = filterExt.getFilteredListOfServers(servers, invocation);
+      if(filterExt.enabled()) {
```

Comment:
```
if(filterExt.enabled){continue;}
xxxx
```
This is better, or logs is not corret when servers is empty.

Reply:
done